### PR TITLE
feat : Change Year in footer

### PR DESCRIPTION
**Pull request by Ayush7614**
This Pr Contains

* Change Year in footer of website

### CODE REVIEW

**Comment by Ayush7614** [comment]
CC : @leecalcote @warunicorn19

**Comment by warunicorn19** [comment]
@Ayush7614 thanks for this, any chance you could make this dynamic? So that we don't have to go change the year every new year.

**Comment by l5io** [comment]
🚀 Preview for commit 443b819078e897edf6fcc3478ec57d4efaec5d38 at: https://62167fa54224ea1d75cd73c2--layer5ng.netlify.app

**Comment by leecalcote** [comment]
Thanks for this update, @Ayush7614. 👍

Two requests to complete prior to this PR being merged:

1. Please incorporate @warunicorn19's feedback as mandatory behavior. 
1. Please resync your fork with master so that you do not include other people's commits in your PR. 

Let us know if you're unable to complete either of these two items within a week's timeframe.

**Review by leecalcote — CHANGES_REQUESTED** [review]
Changes requested

**Comment by Ayush7614** [comment]
> Thanks for this update, @Ayush7614. 👍
> 
> Two requests to complete prior to this PR being merged:
> 
>     1. Please incorporate @warunicorn19's feedback as mandatory behavior.
> 
>     2. Please resync your fork with master so that you do not include other people's commits in your PR.
> 
> 
> Let us know if you're unable to complete either of these two items within a week's timeframe.

Sure @leecalcote 
@warunicorn19 please check

**Comment by YashKamboj** [comment]
@Ayush7614 You can easily make it dynamic like this.
`<script>document.write(new Date().getFullYear())</script>`

**Comment by leecalcote** [comment]
Recheck? Fork is still out of sync. And date still isn't dynamic.

File: src/assets/discuss/html/footer.html [hunk 1]
```diff
@@ -88,7 +88,7 @@ <h5 class="footer-h5"><a href="https://layer5.io/company/contact">Contact</a></h
         <div class="footer-bottom row">
             <hr class="footer-hr" />
             <div class="footer-bottom-left">
-                <p>2021 Copyright ©Layer5, Inc | All Rights Reserved</p>
+                <p>2022 Copyright ©Layer5, Inc | All Rights Reserved</p>
```
Comment (YashKamboj) [inline src/assets/discuss/html/footer.html:91]: @Ayush7614 You can easily make it dynamic like this.
<script>document.write(new Date().getFullYear())</script>

**Review by YashKamboj — CHANGES_REQUESTED** [review]

File: src/assets/discuss/html/footer.html (hunk 1 above)
Comment (Chadha93) [reply src/assets/discuss/html/footer.html:91]: @Ayush7614 updates? you can directly use @YashKamboj 's approach to implement this

**Comment by l5io** [comment]
🚀 Preview for commit 300358c5fa2a6fea751f39675b98781c272edabf at: https://621b9e41ae1b82d2e239a62a--layer5ng.netlify.app

File: src/assets/discuss/html/footer.html (hunk 1 above)
Comment (leecalcote) [reply src/assets/discuss/html/footer.html:91]: @Ayush7614 is this something that you can do?

**Comment by l5io** [comment]
🚀 Preview for commit 21443e141a211225be091cc802bb79534b565e94 at: https://621ff1cc41909f8dbb627e6f--layer5ng.netlify.app

**Comment by YashKamboj** [comment]
@Ayush7614 Do you need help with it???

**Comment by Abhijay007** [comment]
@Ayush7614 are you working on this PR?

**Comment by leecalcote** [comment]
Thanks for opening this PR, @Ayush7614. It looks like another contributor has added a dynamic fetch to resolve this issue. We'll go ahead and close this out. I encourage you to pick up another issue and keep contributing…

**Pull request closed without merging**